integration-tests/actions/ocr2_helpers_local.go

```diff
@@ -272,3 +274,38 @@ func GetOracleIdentitiesWithKeyIndexLocal(
 
 	return S, oracleIdentities, eg.Wait()
 }
+
+func DeleteAllOCR2JobsAndBridges(nodes []*client.ChainlinkClient) error {
```

Comment:
@george-dorin `And` in a function name is always a bit sus and in this case it feels like we're coupling the two (deleting a job and its bridges) together just to save an additional O(N) loop. Could this be separated out?

Reply:
Split the function in https://github.com/smartcontractkit/chainlink/pull/9934/commits/1c93b74da44a3c243581dbec924326a9f91ad077